packages/calypso-analytics/global.d.ts

```diff
@@ -0,0 +1,7 @@
+export {};
+declare global {
+	interface Window {
+		_tkq: Array< Array< any > >;
+		BUILD_TIMESTAMP: string;
```

Comment:
Is this intended to be a general purpose library? `BUILD_TIMESTAMP` global is very Calypso-specific, isn't it?

Reply:
hmm yes, a more general approach would be to get the caller to pass in `BUILD_TIMESTAMP`... but that's not very nice for useability, I think it is better if we do just try to read the `window.BUILD_TIMESTAMP` if it's there, just to keep the function easy to use at the expense of leaking calypso concepts into the analytics library. What do you think?